src/Activity.js

```diff
@@ -32,9 +32,5 @@ class Activity {
     this.completed = true;
   }
   saveToStorage() {
-  var stringifiedActivity = stringifyActivities(savedActivities);
```

Comment:
I believe it was a smart decision for us to refactor and remove the localStorage section of the project and not dive into another iteration this close to the deadline. Let's clean up some of our current work and make it look nice and pretty :)